RFCs/platform-practice-4ps.md

```diff
@@ -0,0 +1,82 @@
+    
+-   Avenue for people to bring what they find important: Provide a platform for individuals to contribute their insights and concerns.
+    
+-   Radiate intent of changes in the Platform: Communicate the purpose and intent behind any changes.
+    
+-   Evolving a platform towards a coherent direction: Ensure alignment among engineers and product teams, steering the platform in a cohesive direction.
+    
+
+## Product:
+
+*   Practice notes with the main topics: Document key points and decisions made during the practice.
+*   Possible action items discussed: Identify and outline potential actions for improvement.
+*   Information was shared and people aligned: Ensure information is disseminated and team members are aligned on the discussed topics.
+    
+
+## People:
+
+* It's open to everybody in PDDE and part of the engineering team calendar: Encourage inclusivity by making the practice accessible to all team members.
+
+### Roles of this practice:
+
+#### 1. Facilitator:
+   * The Facilitator plays a crucial role in managing the process of the platform practice. This includes time-keeping, organizing the agenda, reviewing takeaways and action items, and diligently documenting sessions in Notion. The Facilitator ensures the smooth flow and effectiveness of each session.
+
+#### 2. Champion:
+   * The Champion is the driving force behind topics discussed in the platform practice. They bring forth specific subjects with clear objectives, whether it's seeking a decision or fostering an open discussion. The Champion actively guides discussions, aiding the Facilitator in achieving meaningful conclusions for each topic.
+
+#### 3. Curator:
+   * The Curator takes on the responsibility of shaping the overall direction of the platform practice. They provide valuable advice and curate relevant topics that align with the practice's goals. The Curator's role is crucial in maintaining the practice's focus and ensuring that discussions contribute to the overall improvement and coherence of the platform.
+
+
+## Process:
+
+*   Review of previous action items: Begin each session by reviewing the progress of previously discussed action items.
+    
+*   Review the agenda for the day: Outline the topics covered during the practice session.
+    
+*   Presentation or discussion of the topics (time-boxed 10-15 min):
+
+    
+    * Ensure focused discussions within a specified time frame.
+    * Verify if there's a need to continue on the topic: Assess whether additional discussion or action is required for a particular topic.
+    * Another time-box (8-10min): Further discussions or actions within a constrained timeframe.
+    * Wrap-up of the topic: Summarize key points and decisions made during the discussion.
+    
+
+-   Conclusion of the meeting: Recap important takeaways and action items, and formally conclude the practice session (it's ok to not have any action item due to the context of the topics)
```

Comment:
Maybe clarify who leads each of these? With 3 "roles" there's a risk of diffuse responsibility.

Reply:
Updated